deltamail/main.py

```diff
@@ -1,3 +1,57 @@
+#from campaign import CampaignFactory
+
+import argparse
+
+import os
+
 def console_main():
-    print "It works"
+		
+	current=os.getcwd()
+
+	parser = argparse.ArgumentParser(description='Command line mail app \n Choose Type: \n smartSend: pathname which has the required files \n noobSend: pass individual')
+	parser.add_argument('-s','--subject', help='Subject of email')
+	parser.add_argument('-r','--receivers',nargs='+',help='Receivers List in qoutes separated by space')
+	parser.add_argument('-t','--template',help="template file" )
+	parser.add_argument('-g','--globalVar',help="global variable file")
+	parser.add_argument('-sm','--smartSend',help="path for smart send" ,default="NULL")
+
+	args = vars(parser.parse_args())
+
+	smartSend=args['smartSend']
+
+	def smartSendFun():
+
+		subject=smartSend+"\subject.txt"
```

Comment:
If you're trying to join paths use the os.path.join function to ensure that it works on all platforms.